### PR TITLE
Fix documentation of QuadraticCoreAirgap

### PR DESCRIPTION
**Pull request by christiankral**
In #3307 a leakage model was added in parallel to the air gap (and coil) model:

image

Therefore the included image does not match the model as it does not include the leakage effect: 

image

In addition to that the table and the text do not match the model any more:

image

I made an attempt to fix some parts of the documentation. The following issues are still missing:

- @AHaumer Please update the sketch which shows only one dashed line going straight through the airgap
- The included table does currently not show the leakage path as it has not "real" area of cross section and length; @AHaumer I wonder if can somehow overcome this problem or just leave the table as it is

### CODE REVIEW

File: Modelica/Magnetic/FluxTubes/Examples/BasicExamples/QuadraticCoreAirgap.mo [hunk 1]
```diff
@@ -141,20 +141,27 @@ Magnetic field strength times length of the flux line gives magnetic potential d
...
-<tr><td>airgap    </td><td>a*a          </td><td>delta        </td><td>1                 </td><td>flux / cross section</td><td>B/&mu;<sub>0</sub>                   </td><td>H*length         </td></tr>
+<tr><td>airgap    </td><td>a*a          </td><td>delta        </td><td>1                 </td><td>useful flux / cross section</td><td>B/&mu;<sub>0</sub>            </td><td>H*length         </td></tr>
 <tr><td>lower yoke</td><td>a*a          </td><td>l - a        </td><td>&mu;<sub>r</sub>  </td><td>flux / cross section</td><td>B/(&mu;<sub>r</sub>*&mu;<sub>0</sub>)</td><td>H*length         </td></tr>
 <tr><td>total     </td><td>             </td><td>             </td><td>                  </td><td>                    </td><td>                                     </td><td>&Sigma; mmf = N*I</td></tr>
 </table>
 <p>
-Note that since no leakage is present, the magnetic flux is the same in every element - they are connected in series. 
-For calculation of the length of flux lines, a medium flux line (dashed line) is used.
+Note that there is a leakage flux path present. Therefore the total magnetic flux of in core splits into 
+<ul>
```
Comment (beutlich) [inline Modelica/Magnetic/FluxTubes/Examples/BasicExamples/QuadraticCoreAirgap.mo]:
```suggestion
</p>
<ul>
```

File: Modelica/Magnetic/FluxTubes/Examples/BasicExamples/QuadraticCoreAirgap.mo [hunk 2]
```diff
@@ -141,20 +141,27 @@ Magnetic field strength times length of the flux line gives magnetic potential d
...
+<li>the useful flux through the airgap and</li>
+<li>the leakage flux through the leakage element.</li>
+</ul>
```
Comment (beutlich) [inline Modelica/Magnetic/FluxTubes/Examples/BasicExamples/QuadraticCoreAirgap.mo]:
```suggestion
</ul>
<p>
```

**Comment by christiankral** [comment]
@beutlich Thanks for fixing these issues
By the way: I like Markdown a lot more than HTML ...

**Comment by beutlich** [comment]
> By the way: I like Markdown a lot more than HTML ...

Luckily we have HTML Tidy now running through Travis CI.

**Comment by dietmarw** [comment]
@AHaumer There are still the things missing that @christiankral mentioned above. Can you provide the updated sketches and table?

**Comment by AHaumer** [comment]
@dietmarw 
Thanks to @christiankral - he has updated the table already!
I tried to update the sketch accordingly.

**Pull request merged**